pkgs/applications/misc/gcstar/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchFromGitLab
+, perl
+, perlPackages
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcstar";
+  version = "1.7.2";
+
+  src = fetchFromGitLab {
+    owner = "Kerenoc";
+    repo = "GCstar";
+    rev = "v${version}";
+    sha256 = "1vqfff33sssvlvsva1dflggmwl00j5p64sn1669f9wrbvjkxgpv4";
+  };
+
+  perlDeps = [ perl ] ++ (with perlPackages; [
```

Comment:
Grepping nixpkgs, it does not appear that `perlDeps` has any special meaning. Why not just set add these directly to `buildInputs`?

Reply:
Added to `buildInputs`, thanks